pkg/deviceshifu/deviceshifuhttp/deviceshifuhttp.go

```diff
@@ -221,19 +221,27 @@ func (handler DeviceCommandHandlerHTTP) commandHandleFunc() http.HandlerFunc {
 			utils.CopyHeader(w.Header(), resp.Header)
 			w.WriteHeader(resp.StatusCode)
 
+			// Handling deviceshifu stuck when responseBody is a stream
+			_, shouldUsePythonCustomProcessing := deviceshifubase.CustomInstructionsPython[handlerInstruction]
+			if !shouldUsePythonCustomProcessing {
+				klog.Infof("Needn't Custom ")
```

Comment:
let's remove this log for instructions that have not configured custom processing